src/components/RangeSlider/RangeSlider.tsx

```diff
@@ -40,9 +46,15 @@ export const Container = styled.div`
     ${
       disabled
         ? `
-      filter: grayscale(1) contrast(.5) brightness(1.2);
-      pointer-events: none;
-    `
+        filter: grayscale(1) contrast(.5) brightness(1.2);
+        pointer-events: none;`
+        : ''
+    }
+
+    ${
+      isReadOnly
+        ? `
+        pointer-events: none;`
```

Comment:
we want to keep pointer events, so things like "hover to see handle label" still work, but other mouse/kb inputs don't change the values.